### PR TITLE
feat: update slate to ^0.32.0

### LINKED ISSUE #7: Update for current slate version?
Is there an update for newest slate?
`(slate@0.32.0): The kind property of Slate objects has been renamed to object`

### PR DESCRIPTION
**Pull request by bsouthga**
Updates slate to current version. Mostly consists of changing `kind` to `object`. Resolves #7.

### CODE REVIEW

**Comment by tommoor** [comment]
Would you be open to changing the Slate dependency to > 32? As these changes were introduced in 32, not 33.

**Comment by bsouthga** [comment]
👍 , pushed a commit to drop it back to `^0.32.0`

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "repository": "git://github.com/GitbookIO/slate-trailing-block.git",
   "main": "./dist/index.js",
   "peerDependencies": {
-    "slate": "^0.29.0"
+    "slate": "^0.33.0"
```
Comment (SamyPesse) [inline package.json]: It should be `^0.32.0`

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "react": "^15.3.1",
     "react-dom": "^15.3.1",
     "read-metadata": "^1.0.0",
-    "slate": "^0.29.0"
+    "slate": "^0.32.0"
```
Comment (SamyPesse) [inline package.json]: This one could be `^0.33.0`

**Review by SamyPesse — CHANGES_REQUESTED** [review]

**Comment by bsouthga** [comment]
Sorry about that, should have been more careful!

**Review by SamyPesse — APPROVED** [review]
No worries :)

Thank you 🍻

**Comment by SamyPesse** [comment]
Published as `0.5.0` 🚀

**Pull request merged**